src/Harpoon.cpp

```diff
@@ -49,6 +49,16 @@ void Harpoon::RequestChunks() {
    }
 }
 
+
+Harpoon::Battling Harpoon::Cancel() {
+   FreeChunk();
+   RequestChunks();
+   zstr_sendf (mDealer, "<CANCEL>");
```

Comment:
In Harpoon.h, Battling enum includes "CANCEL", but the EnumToString method in this file does not have a case for CANCEL.